core/src/main/scala/com/salesforce/op/stages/impl/preparators/SanityChecker.scala

```diff
@@ -274,18 +274,6 @@ class SanityChecker(uid: String = UID[SanityChecker])
       .toMap[String, Double]
     val numCorrIndices = corrIndices.length
 
-    // build map from indicator group name to any null indicator it may have
-    val nullGroups = for {
-      col <- metaCols
-      if col.isNullIndicator
-        grouping <- col.grouping
-      } yield (grouping, (col, col.index))
-
-    nullGroups.groupBy(_._1).foreach {
-      case (group, cols) =>
-        require(cols.length == 1, s"Vector column $group has multiple null indicator fields: $cols")
```

Comment:
since you are removing this check, also remove this ignored test - https://github.com/salesforce/TransmogrifAI/blob/f37113b4dbbd9078b3315ebfe0ab36975f83493a/core/src/test/scala/com/salesforce/op/stages/impl/preparators/BadFeatureZooTest.scala#L107